charts/accurate/MIGRATION.md

```diff
@@ -0,0 +1,116 @@
+# Migrate from kustomize to Helm
+
+This document describes the steps to migrate from kustomize to Helm.
+
+## Install Helm chart
+
+There is no significant difference between the manifests installed by kusomize and those installed by Helm.
+
+If a resource with the same name already exists in the Cluster, Helm will not be able to create the resource.
+
+```console
+$ helm repo add accurate https://cybozu-go.github.io/accurate/
+$ helm repo update
+$ helm install --namespace accurate accurate accurate/accurate
+Error: rendered manifests contain a resource that already exists. Unable to continue with install: ServiceAccount "accurate-controller-manager" in namespace "accurate" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "accurate"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "accurate"
+```
+
+Before installing Helm chart, you need to manually delete the resources.
```

Comment:
Subnamespaces and namespaces will not be deleted, right?
I want you to write it down so readers won't worry about it.

Reply:
Thanks, updated: https://github.com/cybozu-go/accurate/pull/11/commits/f554b28a970be9f574b4b3b880590153dc6cad88